bin/get-resources

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function usage() {
+  echo "Usage: $0 [options]"
+  echo "Options:"
+  echo "   -n|--namespace <value>"
+  echo "                  Mandatory: specify the namespace to monitor resources (default: all resources)"
```

Comment:
Can we default this to `CF_OPERATOR_NAMESPACE`?

Reply:
sure, all good points, thanks. I will add the requested changes via another commit.